src/BloomBrowserUI/lib/jquery.easytabs.js

```diff
@@ -48,7 +48,7 @@
     tabActiveClass: "active",
     defaultTab: "li:first-child",
     animationSpeed: "normal",
-    tabs: "> ul > li",
+    tabs: "> nav > ul > li",
```

Comment:
changing a library is dangerous. It's so easy for us to update to a newer version of the library and then wonder why things broke. Is there another way to accomplish this?